benchmarks/InitBenchmark.php

```
@@ -31,7 +31,7 @@ public function benchInit(array $params): void {
     Environment::init((bool) $params['contextualize']);
```

PHPMD (2.15.0)

Avoid using static access to class '\DrevOps\EnvironmentDetector\Environment' in method 'benchInit'. (undefined)

(StaticAccess)